openml-utils/src/main/java/com/feedzai/openml/util/validate/ClassificationValidationUtils.java

```diff
@@ -70,4 +76,33 @@ public static void validateClassificationModel(final DatasetSchema schema, final
             throw new ModelLoadingException(msg, e);
         }
     }
+
+    /**
+     * Validates that the model to load can be used with the given parameters.
+     *
+     * @param modelLoader An instance responsible for instantiating models.
+     * @param modelPath   Path of the model.
+     * @param schema      Schema of the model.
+     * @param params      Collection of parameters and the corresponding values.
+     * @throws ModelLoadingException If the path and/or schema are not valid.
```

Comment:
This is actually throwing exception if any of the parameters is not valid; is that your intention? It's not clear to me if this bit of code was really needed here.

Reply:
yes it was my intention, this method was created to be used by every model provider. In that method we want to throw a ModelLoadingException if the validation fails. Otherwise it would be similar to MachineLearningModelLoader#validateForLoad()